ios/Classes/core/PMManager.m

```diff
@@ -714,7 +721,7 @@ - (NSString *)makeAssetOutputPath:(PHAsset *)asset
     } else {
         filename = [asset valueForKey:@"filename"];
     }
-    filename = [NSString stringWithFormat:@"%@%@_%@", modifiedDate, isOrigin ? @"_o" : @"", filename];
+    filename = [NSString stringWithFormat:@"%@_%@%@_%@", id, modifiedDate, isOrigin ? @"_o" : @"", filename];
```

Comment:
Would it be too long if we added the ID here?

Reply:
Other suggestion?

The id of asset will add 44 chars.
The biggest of darwin filename is 255.

Reply:
Maybe compress the ID into 6~8 characters ID?

Reply:
I'm not sure what is the probability that a certain substring of the id will be repeated.